### ISSUE TITLE
[ENDPOINT] Proposed design change for getShopReceipts

### ISSUE THREAD

**Issue by timalin** [etsy/open-api#161]
Please consider option to include Transactions when calling for shop receipts.
Please also include product_data(as it was in APIv2) in Transactions.

Currently if I want to know what products where purchased I need to call receipts, then call transactions of the receipt. And even then it only tells me the product_id, which I would need to associate with products separately and can't actually be used to edit listing.

Please tell me if I'm missing something.

**Comment by gemini133** [comment]
couldn't agree more!

**Comment by breenyoung** [comment]
I brought this up earlier this year in https://github.com/etsy/open-api/issues/97

Without these associations the calls needed to get this data increase in what used to be a single call per receipt

**Comment by gemini133** [comment]
sigh, they don't response this trouble directly

**Comment by etsyachristensen** [comment]
@timalin , @breenyoung , @gemini133 - Rest assured, this is on our list. We've been short staffed lately with summer vacations and such. Some turn over in teams as well. We have some hot items that are on the top of the pile and I believe that transactions are among them. Sorry we haven't been responding terribly quickly to the posts, but please know we hear you and we are working to get through many of these requests as quickly as possible.

**Comment by mtj928** [comment]
Agree 100%, +1

in addition it would be helpful if the documentation for get receipt transactions clarified that there apparently is no longer a limit on the number of transactions returned, from my limited testing the transactions returned are no longer limited only to the V2 default of 25.

**Comment by WheelerCC** [comment]
This looks to have been changed now. At least transactions are now returned by getShopReceipts. Each transaction still only contains the product_id, however. No SKU, so an additional call is still required.

https://developer.etsy.com/documentation/reference#operation/getShopReceipts

**Comment by breenyoung** [comment]
> Looks like this change has been made. Reflected by the documentation as well as my own usage. Each transaction still only contains the product_id, however. No SKU, so an additional call is still required. :(
> 
> https://developer.etsy.com/documentation/reference#operation/getShopReceipts

Yeah I see that, not quite there yet. @etsyachristensen any chance the transactions returned will also themselves contain a product association for them so we don't need N+1 extra calls on transactions to get the product info?

**Comment by breenyoung** [comment]
Thanks for the prompt replies @etsyachristensen !
